core/src/test/scala/unit/kafka/server/SaslApiVersionsRequestTest.scala

```diff
@@ -52,7 +50,7 @@ class SaslApiVersionsRequestTest extends BaseRequestTest with SaslSetup {
 
   @Test
   def testApiVersionsRequestBeforeSaslHandshakeRequest(): Unit = {
-    val plaintextSocket = connect(protocol = securityProtocol)
+    val plaintextSocket = connect()
```

Comment:
Maybe we should rename this to `socket` as the current name is a bit misleading. Same for the other methods.